src/collar/oc_leash.lsl

```diff
@@ -558,6 +558,8 @@ UserCommand(integer iAuth, string sMessage, key kMessageID, integer bFromMenu) {
                 if (llGetAgentSize((key)sVal)) g_iPassConfirmed = FALSE;
                 else g_iPassConfirmed = TRUE;
                 LeashTo((key)sVal, kMessageID, iAuth, lPoints, FALSE);
+                
+                llSay(0x89a, llList2Json(JSON_OBJECT, ["type", "leash", "name", sVal]));
```

Comment:
I'd prefer to have this channel constant declared up top and then used by name here.  Also a base-10 representation would be more readable to most scripters (to me, at least :).